pkg/quota/quota.go

```diff
@@ -33,27 +35,41 @@ type BaseQuota interface {
 func NewQuota(name string) BaseQuota {
 	var quota BaseQuota
 	switch name {
-	case "grp":
+	case "grpquota":
 		quota = &GrpQuota{
 			quotaIDs:    make(map[uint32]uint32),
 			mountPoints: make(map[uint64]string),
 		}
-	case "prj":
+	case "prjquota":
 		quota = &PrjQuota{
 			quotaIDs:    make(map[uint32]uint32),
 			mountPoints: make(map[uint64]string),
 			devLimits:   make(map[uint64]uint64),
 		}
 	default:
-		quota = &GrpQuota{
-			quotaIDs:    make(map[uint32]uint32),
-			mountPoints: make(map[uint64]string),
+		kernelVersion, err := kernel.GetKernelVersion()
+		if err == nil && kernelVersion.Major >= 4 {
+			quota = &PrjQuota{
+				quotaIDs:    make(map[uint32]uint32),
+				mountPoints: make(map[uint64]string),
+				devLimits:   make(map[uint64]uint64),
+			}
+		} else {
+			quota = &GrpQuota{
+				quotaIDs:    make(map[uint32]uint32),
+				mountPoints: make(map[uint64]string),
+			}
 		}
 	}
 
 	return quota
 }
 
+// SetQuotaDriver is used to set global quota driver.
+func SetQuotaDriver(name string) {
```

Comment:
this function is not used, do you mean to use it in line 19?

Reply:
It has used at main.go:130

Reply:
It has used at main.go:130